dragos-worldview-threat-intelligence/info.json

```diff
@@ -361,23 +362,24 @@
         },
         {
           "title": "Serials",
-          "tooltip": "CSV/list of serials",
-          "description": "Specify CSV/list of serial for which products you want to retrieve. Search for products matching a report serial.",
+          "tooltip": "List of report serial numbers",
+          "description": "Specify the list of report serial numbers for which you wish to retrieve the details. Search for reports matching the specified serial(s).",
           "required": false,
           "editable": true,
           "visible": true,
           "type": "text",
           "name": "serials",
-          "placeholder": "serial1, serial2"
```

Comment:
Please make change in placeholder with "For e.g. serial1, serial2, etc".  Also, make changes wherever use placeholder key.

Reply:
Placeholders updated as per recommendation.